rtl_433_mqtt_autodiscovery/Dockerfile

```diff
@@ -1,24 +1,15 @@
-ARG BUILD_FROM
-FROM $BUILD_FROM
-
-ENV LANG C.UTF-8
-
-MAINTAINER pbkhrv@pm.me
-
-# Install latest pip
-RUN python3 -m ensurepip && \
-    pip3 install --no-cache --upgrade pip
-
-# Install rtl_433 mqtt bridge prereq
-RUN pip3 install --no-cache paho-mqtt
-
-# Clone rtl_433 repo and copy the latest tested version of the mqtt bridge script
-# (latest as of 12/25/2020)
-RUN curl -LO https://github.com/pbkhrv/rtl_433/raw/29c04d8b412eb09f4cd1db4b363050d2d72e9065/examples/rtl_433_mqtt_hass.py && \
-  chmod +x /rtl_433_mqtt_hass.py
-
-# Run script
-COPY run.sh /
-RUN chmod a+x /run.sh
+ARG BUILD_FROM=homeassistant/amd64-base-python:3.9-alpine3.13
```

Comment:
I left it in as a default for the argument (it is usually overwritten by the deployment). That style was copied from another Add-on. I'm indifferent if you want to remove it, I just like keeping a default argument there in case someone tries to build the Dockerfile not from CI.

Reply:
If you found it in another addon I missed, then that's fine. Thanks!